<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartProgramSetting</class>
 <widget class="QWidget" name="SmartProgramSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>417</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Smart Program Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>System</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="CB_PreventUpdate">
        <property name="text">
         <string>Prevent Game Update after Restart (BDSP only)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="minimumSize">
           <size>
            <width>115</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Date Arrangement:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="CB_DateArrangement">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>JP (yyyy/mm/dd)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EU (dd/mm/yyyy)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>US (mm/dd/yyyy)</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_13">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Switch's Current Date:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="CurrentDate">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumDate">
           <date>
            <year>2060</year>
            <month>12</month>
            <day>31</day>
           </date>
          </property>
          <property name="minimumDate">
           <date>
            <year>2000</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
          <property name="displayFormat">
           <string>yyyy/MM/dd</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PB_CurrentDate">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Set Today's Date</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GB_Sound">
     <property name="title">
      <string>Sound</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="RB_DefaultSound">
          <property name="text">
           <string>Default Sound</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RB_CustomSound">
          <property name="text">
           <string>Custom Sound</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PB_PlaySound">
          <property name="text">
           <string>Play Sound</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Custom Sound:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_Sound">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="PB_Sound">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GB_Stream">
     <property name="title">
      <string>Stream Counter</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Each program will export individual text files for
each stat in &quot;Current Stats&quot; in &quot;StreamCounters&quot; folder.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CB_ExcludePrefix">
        <property name="text">
         <string>Exclude Stat Prefix (Show Numbers Only)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Others</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QCheckBox" name="CB_LogSave">
        <property name="text">
         <string>Auto-save log for each Smart Program session</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CB_LogDebugCommands">
        <property name="text">
         <string>DEBUG: Show all commands</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CB_LogDebugColor">
        <property name="text">
         <string>DEBUG: Show color matching tests</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
